.pre-commit-config.yaml

```
@@ -0,0 +1,95 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.11.4
```

@madsbk I think this update is needed to fix style checks. I’ll commit it.
```suggestion
    rev: 5.12.0
```